src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditQuestionsUiTest.java

```diff
@@ -81,111 +79,65 @@ public void allTests() throws Exception{
         testEditQuestionLink();
         testEditQuestionAction();
         
-        // No testDeleteQuestionAction() method as this is tested separately
-        // for each of the question-types later
-        
         testGetQuestionLink();
-
-        testTwoNewEssayQuestionsWithRecipientBeingNobodySpecific();
-
+        testCopyQuestion();
+        
+        testChangeFeedbackRecipient();
+        
+        testEditQuestionNumberAction();
+        
+        testDeleteQuestionAction(2);
+        testDeleteQuestionAction(1);
+    }
+    
+    @Test
```

Comment:
MCQ/MSQ and other question types are supposed to be in separate test files, because each one takes a long time to complete.
They should be split into files similar to FeedbackRubricQuestionUiTest.